uber/react-digraph
# Only import the expand icon

Only import font-awesome `expand` icon instead of the entire `react-icons` library. 
This optimization reduces the production build output by 374k, a 68% reduction in asset size.

**How to test this**:
1. checkout the branch locally
2. install dependency changes via `yarn` or `npm install`
3. run `yarn example` and verify everything looks normal
4. merge https://github.com/uber/react-digraph/pull/107 and run `yarn analyze-bundle` to see prod build stats.

Visualized improvements via `webpack-build-analyzer`:

<img width="1439" alt="screen shot 2019-02-24 at 8 35 41 am" src="https://user-images.githubusercontent.com/2060731/53307130-1f936700-384a-11e9-8e61-6fab70a69f9f.png">
<img width="1439" alt="screen shot 2019-02-24 at 8 29 26 am" src="https://user-images.githubusercontent.com/2060731/53307133-23bf8480-384a-11e9-9021-5f6e4f66f4e8.png">


Resolved #108
